apps/builder/app/builder/features/props-panel/control.tsx

```diff
@@ -21,15 +21,7 @@ import { ImageManager } from "~/builder/shared/image-manager";
 import type { UserPropValue } from "./use-props-logic";
 import type { SetProperty } from "../style-panel/shared/use-style-data";
 
-const textControlTypes = [
-  "text",
-  "array",
-  "color",
-  "date",
-  "file",
-  "number",
-  "object",
-] as const;
+const textControlTypes = ["text", "color", "number", "multilineText"] as const;
```

Comment:
Original names are taken from storybook's controls addon that tries to map data types to control names

https://storybook.js.org/docs/react/essentials/controls#annotation
Should we stick with those and add ours on top?

Reply:
They just agreed to add multiline-text

https://discord.com/channels/486522875931656193/689023212519948302/1075001187776069682

Reply:
Why do we need any kind of compatibility with what storybook is doing? I feel like this tights our hands and will probably require me an extra day just to understand what kind of compatibility we want

Reply:
I know we are not using stories right now, but we will need a way to use them some day.

The basic idea was to be able to start webstudio using cli just like storybook and let it pick-up the stories

Reply:
> Original names are taken from storybook's controls addon that tries to map data types to control names

I can't see a list similar to our original names anywhere at the link.

In the table I'm not sure what "Data Type" is. We don't get types in this format from the library that we use in `generate-arg-types`. For example, instead of `array` it would give us something like `string[]`

Reply:
I've added all types of controls from storybook to our system. Some of them we neither generate nor support in props panel. But now it's clear in code.

Reply:
Also created this issue: https://github.com/webstudio-is/webstudio-builder/issues/1036